blog/index.tt

```diff
@@ -1,6 +1,405 @@
 
```

Comment:
I'm a bit confused. I don't remember how this gets updated. But I assume it shouldn't be committed here.

I also assume this should be emptied (out of scope of this PR) since it gets updated automatically from external sources.